docs/notebooks/zero_inflated_regression.ipynb

```diff
@@ -0,0 +1,1905 @@
+{
```

Comment:
<blockquote> A zero-inflated model assumes that zero counts result from a mixture of two distributions</blockquote><p>This is true for both models. The difference in the ZIPoisson is that the zeroes can come from any of the components, while in the Hurdle Poisson they come only from one of the processes. You get this second thing right in your text, but I think you can be a little bit more explicit for the first. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/725/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>